src/python/zquantum/qubo/conversions.py

```diff
@@ -116,5 +140,10 @@ def convert_measurements_to_sampleset(
         )
     if bqm.vartype != dimod.BINARY:
         raise TypeError("BQM needs to have vartype BINARY")
-    return SampleSet.from_samples_bqm(measurements.bitstrings, bqm)
 
+    bitstrings = [
+        [int(change_bitstring_convention != bit) for bit in bitstring]
```

Comment:
Should bitstrings be a list of lists or a list of tuples? Everything else looks great.

Reply:
Ok, I've changed it in https://github.com/zapatacomputing/z-quantum-qubo/pull/11/commits/17aaec5d60ddde257453ab6d230fa633bcf1dac2